dependencyManagement/build.gradle.kts

```diff
@@ -113,7 +113,7 @@ rootProject.allprojects {
 
 fun isNonStable(version: String): Boolean {
   val stableKeyword = listOf("RELEASE", "FINAL", "GA").any { version.toUpperCase().contains(it) }
-  val regex = "^[0-9,.v-]+(-r)?$".toRegex()
+  val regex = "^[0-9,.v-]+(-r|-alpha)?$".toRegex()
```

Comment:
General Question: What prompted this change?

Reply:
The task that detected changes in dependencies was ignoring some packages because they had this `alpha` suffix. I checked and the packages existed for a long time. It is not something new. I think we were updating the versions manually without paying attention because those were opentelemetry packages that required update anyways.